From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jaime Dias <jaimejosediasnt@gmail.com>
Date: Fri, 14 Aug 2026 14:35:00 -0300
Subject: [PATCH 17/17] usb2otg: enable bring-up diagnostics

Temporarily enable the driver's existing diagnostics while the first modular
OpenDevice on Bellatrix is being brought up. Remove this patch once the failing
initialization stage has been identified and given a focused diagnostic.
---
 arch/arm-native/soc/broadcom/2708/usb/usb2otg/usb2otg_device.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm-native/soc/broadcom/2708/usb/usb2otg/usb2otg_device.c b/arch/arm-native/soc/broadcom/2708/usb/usb2otg/usb2otg_device.c
index 1111111111..2222222222 100644
--- a/arch/arm-native/soc/broadcom/2708/usb/usb2otg/usb2otg_device.c
+++ b/arch/arm-native/soc/broadcom/2708/usb/usb2otg/usb2otg_device.c
@@ -4,7 +4,7 @@
     Copyright (C) 2013-2026, The AROS Development Team. All rights reserved.
 */
 
-#define DEBUG 0
+#define DEBUG 1
 #include <aros/debug.h>
 #include <aros/atomic.h>
 
-- 
2.43.0
